### PR TITLE
Add new board: Cytron Maker Zero SAMD21

### PR DESCRIPTION
**Pull request by mlhakimz**
Hello, can you help to add this new board?

[https://my.cytron.io/p-maker-zero-samd21](https://my.cytron.io/p-maker-zero-samd21)

Thanks.

### CODE REVIEW

File: ports/atmel-samd/boards/cytron_maker_zero_samd21/pins.c [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    { MP_ROM_QSTR(MP_QSTR_A0), MP_ROM_PTR(&pin_PA02) },
+    { MP_ROM_QSTR(MP_QSTR_A1), MP_ROM_PTR(&pin_PB08) },
+    { MP_ROM_QSTR(MP_QSTR_A2), MP_ROM_PTR(&pin_PB09) },
+    { MP_ROM_QSTR(MP_QSTR_A3), MP_ROM_PTR(&pin_PA04) },
+    { MP_ROM_QSTR(MP_QSTR_A4), MP_ROM_PTR(&pin_PA05) },
+    { MP_ROM_QSTR(MP_QSTR_A5), MP_ROM_PTR(&pin_PB02) },
+
+    { MP_ROM_QSTR(MP_QSTR_D0), MP_ROM_PTR(&pin_PA11) },
+    { MP_ROM_QSTR(MP_QSTR_RX), MP_ROM_PTR(&pin_PA11) },
```
Comment (dhalbert) [inline ports/atmel-samd/boards/cytron_maker_zero_samd21/pins.c:14]: Put the preferred pin name first, if `RX` is preferred. Same for other pins.

**Review by dhalbert — APPROVED** [review]
This is fine but one comment about the ordering of the pin names, if you want to change it.

File: ports/atmel-samd/boards/cytron_maker_zero_samd21/pins.c (hunk 1 above)
Comment (dhalbert) [reply ports/atmel-samd/boards/cytron_maker_zero_samd21/pins.c:14]: let me know if you don't want to change it and I will merge

Comment (mlhakimz) [reply ports/atmel-samd/boards/cytron_maker_zero_samd21/pins.c:14]: Okay, but I think the arrangement is fine for now. Thanks.

**Comment by mlhakimz** [comment]
@dhalbert I'm keeping the pins order for now. Please merge the PR. Thanks again.

**Pull request merged**